pkg/NETStandard.Library.NETFramework/NETStandard.Library.NETFramework.pkgproj

```diff
@@ -96,7 +96,31 @@
     </ItemGroup>
   </Target>
 
-  <Target Name="ExpandClosureFilesByTargetFramework" 
+  <Target Name="AddFrameworkClosureFiles" AfterTargets="ExpandClosureFilesByTargetFramework">
+
+    <ItemGroup>
+      <FrameworkClosureFile Include="@(NetFxReference->'$(RefRootPath)netfx\%(Identity).dll')" >
+        <FileSet>reference-net461</FileSet>
+      </FrameworkClosureFile>
+      <FrameworkClosureFile Include="@(NetFxReference->'$(RefRootPath)netfx\%(Identity).dll')" >
+        <FileSet>runtime-net461</FileSet>
+      </FrameworkClosureFile>
+      <FrameworkClosureFile Include="@(NetFxReference->'$(RefRootPath)net462\%(Identity).dll')" >
+        <FileSet>reference-net462</FileSet>
+      </FrameworkClosureFile>
+      <FrameworkClosureFile Include="@(NetFxReference->'$(RefRootPath)net462\%(Identity).dll')" >
+        <FileSet>runtime-net462</FileSet>
+      </FrameworkClosureFile>
+      <FrameworkClosureFile Include="@(NetFxReference->'$(RefRootPath)net47\%(Identity).dll')" >
+        <FileSet>reference-net47</FileSet>
+      </FrameworkClosureFile>
+      <FrameworkClosureFile Include="@(NetFxReference->'$(RefRootPath)net47\%(Identity).dll')" >
```

Comment:
Since I commented about slashes earlier, should these be front- or back- slashes?